src/node_url.cc

```diff
@@ -131,11 +131,13 @@ namespace url {
   static int ToUnicode(std::string* input, std::string* output) {
     output->reserve(input->length());
     *output = input->c_str();
```

Comment:
Yeah, seems safe, I just wasn't sure of the original intent of `*output = input->c_str()`.